src/commands/commerce/store/quickstart/setup.ts

```diff
@@ -116,13 +117,19 @@ export class StoreQuickstartSetup extends SfdxCommand {
         // Copy all example files
         FILE_COPY_ARGS.forEach((v) => modifyArgFlag(v.args, v.value, this.argv));
         await FilesCopy.run(addAllowedArgs(this.argv, FilesCopy), this.config);
+
         this.devHubUsername = (await this.org.getDevHubOrg()).getUsername();
         this.statusFileManager = new StatusFileManager(
             this.devHubUsername,
             this.org.getUsername(),
             this.flags['store-name'] as string
         );
         const storeType = StoreQuickstartSetup.getStoreType(this.org.getUsername(), this.flags, this.ux, this.logger);
+        const storeTemplate = this.getTemplateName();
+
+        // LWR uses DigitalExperienceBundle, B2B Aura uses ExperienceBundle.
+        this.isUsingDigitalExperienceBundle = storeType !== 'B2B' || storeTemplate !== 'B2B Commerce (Aura)';
```

Comment:
Don't B2B LWR use DigitalExperienceBundle?

Reply:
It does. I also tried to said that in the comment but I can be clearer.

Reply:
so for B2B `isUsingDigitalExperienceBundle` will be false right? I think just `storeTemplate !== 'B2B Commerce (Aura)` is enough

Reply:
For B2B LWR, `isUsingDigitalExperienceBundle` will be true.